indispensable-josef/build.gradle.kts

```diff
@@ -46,6 +46,7 @@ kotlin {
 
 exportIosFramework(
     "IndispensableJosef",
+    transitiveExports=false,
```

Comment:
same question

Reply:
removing the need to manually declare all transitive api dependencies as exports. effects multibase and encoding, mostly